src/App.js

```diff
@@ -1,10 +1,23 @@
 import React, { Component } from 'react';
 import Helmet from 'react-helmet';
 import './App.css';
+import Toggle from './components/CommonComponents/Toggle';
```

Comment:
Can you just rename the CommonComponents component to `RefreshToggle` please? Shouldn't need the CommonComponent.

Reply:
Sure, will do.